pages/Product/components/Reviews/components/List/components/Review/components/Rating/style.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { css } from 'glamor';
+import colors from 'Styles/colors';
+import variables from 'Styles/variables';
+
+const ratingLine = css({
+
+}).toString();
+
+const stars = css({
+  display: 'inline-block',
+  top: 3,
```

Comment:
Consider using the <Grid> component to align elements horizontally and vertically.
This is a better approach than introducing magic numbers.

Reply:
Adjusted line-height.
Done.